worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageProcessorImpl.java

```diff
@@ -88,11 +91,13 @@
      */
     public CoverageProcessorImpl(CoverageService coverageService, BFDClient bfdClient,
                                  @Qualifier("patientCoverageThreadPool") ThreadPoolTaskExecutor executor,
-                                 @Value("${coverage.update.max.attempts}") int maxAttempts) {
+                                 @Value("${coverage.update.max.attempts}") int maxAttempts,
+                                 Ab2dEnvironment ab2dEnvironment) {
         this.coverageService = coverageService;
         this.bfdClient = bfdClient;
         this.executor = executor;
         this.maxAttempts = maxAttempts;
+        this.ab2dEnvironment = ab2dEnvironment;
```

Comment:
May not need to pass this in since it is being used by the callable which should not use it either.